app/src/main/java/com/kickstarter/ui/fragments/BackingFragment.kt

```diff
@@ -364,7 +364,7 @@ class BackingFragment : BaseFragment<BackingFragmentViewModel.ViewModel>() {
     }
 
     private fun setPledgeStatusText(pledgeStatusData: PledgeStatusData) {
-        val ksString = this.viewModel.ksString
+        val ksString = requireNotNull(this.viewModel.ksString)
```

Comment:
requireNotNull will crash if KSString is null, probably safest to unwrap it as optional